iocage_lib/ioc_create.py

```diff
@@ -374,7 +374,9 @@ def _create_jail(self, jail_uuid, location):
                         config['vnet'] = 'on'
 
                     rtsold_enable = 'YES'
-            elif key == 'dhcp' and value == 'on':
+            elif (key == 'dhcp' and value == 'on') or (
+                key == 'ip4_addr' and 'DHCP' in value
```

Comment:
Requiring DHCP uppercase seems odd. Is there anything which forces DHCP to be set as uppercase?

Perhaps

'dhcp' in value.lower() would make sense?